Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1808,6 +1808,7 @@ void MediaPlayerPrivateGStreamer::handleMessage(GstMessage* message)
         if (!m_isLegacyPlaybin && currentState == GST_STATE_PAUSED && newState == GST_STATE_PLAYING)
             playbin3SendSelectStreamsIfAppropriate();
         updateStates();
+        checkPlayingConsitency();
```

Comment:
The bug title has MSE tag so maybe do this only for `isMediaSource()`?

Also, there is a typo in the method name.

Reply:
Whatever... as long as this doesn't impact perf...